xeno/shields.md

```diff
@@ -0,0 +1,112 @@
+## Abstract
+
+<!-- An abstract is a short blurb, about a paragraph or two, succinctly describing your feature. This should mostly be "why", but can include "what". -->
+
+Reworking shields is something that's been requested for a long time. It primarily involves reworking shields shields on all castes (other than rav and crusher) and replacing them with other abilities or miscellaneous balance changes. There are a variety of detailed reasons explained in the reasons section below.
```

Comment:
Don't mention reworking or anything like that. Just describe how shields should actually be in the game.
Do it in present tense, because if this gets merged then that's how it's going to be